temporal-serviceclient/src/main/java/io/temporal/serviceclient/WorkflowServiceStubsImpl.java

```diff
@@ -168,7 +169,9 @@ public WorkflowServiceStubsImpl(
   }
 
   private ScheduledExecutorService startConnectionBackoffResetter(Duration backoffResetFrequency) {
-    ScheduledExecutorService executor = Executors.newSingleThreadScheduledExecutor();
+    ScheduledExecutorService executor =
+        Executors.newSingleThreadScheduledExecutor(
+            new ThreadFactoryBuilder().setDaemon(true).build());
```

Comment:
I would also set the nameFormat property to distinguish these threads in the dumps.

Reply:
Good point, let me do that too.